assets/src/scss/components/_share-buttons.scss

```
@@ -3,59 +3,57 @@
 
   .share-btn {
     border: none;
-    color: $white;
     font-size: $font-size-md;
     opacity: 0.9;
-    width: 4rem;
+    width: 3rem;
     text-align: center;
     line-height: 2.7rem;
-    height: 2.7rem;
-    border-radius: 4px;
+    height: 3rem;
+    border-radius: 50%;
     margin-bottom: 0;
+    background: $share-buttons-bg-color;
 
     &:not(:last-child) {
       margin-inline-end: 8px;
     }
 
     .icon {
-      top: -2px;
+      top: -1px;
+    }
+
+    &:hover,
+    &:active {
+      background: $share-buttons-bg-color__hover;
     }
   }
 
   .twitter {
-    background: $twitter;
+    color: $twitter;
 
-    &:hover,
-    &:active {
-      background: darken($twitter, 20%);
+    .icon {
+      top: 1px;
     }
   }
 
   .facebook {
-    background: $facebook;
+    color: $facebook;
 
-    &:hover,
-    &:active {
-      background: darken($facebook, 20%);
+    .icon {
+      top: 1px;
     }
   }
 
   .email {
```

If `.twitter`, `.facebook` and `.email` share the same `.icon` `top` position, why don't do sth like this?

```suggestion
.twitter, .facebook, .email {
   .icon {
      top: 1px;
    }
}
```
Or directly define this value by default and only change it within [here](https://github.com/greenpeace/planet4-master-theme/pull/1930/files#diff-d411e2d6f82e71296b64bf44ffc623d080796c1ac412983d15ba12a7565d33a3R21) and [here](https://github.com/greenpeace/planet4-master-theme/pull/1930/files#diff-d411e2d6f82e71296b64bf44ffc623d080796c1ac412983d15ba12a7565d33a3R59).

---

Thank you for the feedback.